robotouille_wrapper.py

```diff
@@ -73,6 +73,9 @@ def _state_update(self):
                     if state >= max_num_cuts:
                         literal = pddlgym_utils.str_to_literal(f"iscut({item}:item)")
                         state_updates.append(literal)
+                        if item_name == "onion" or item_name == "potato":
```

Comment:
since we went the PDDL route, let's remove this since it isn't necessary anymore.